### ISSUE TITLE
[IntelliJ-ThirdParty][A11yMAS][Accessibility][ColourContrast][MAS1.4.3][IntelliJ][Wipro]: Colour contrast ratio is below 4.5:1 for selected plugins in settings window.

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/azure-tools-for-java#1381]
**Repro Steps:**

1. Open and Start IntelliJ IDEA.
2. Navigate and select Settings Menu item under File menu.
3. In Settings dialog  box, navigate and select Plugins,
4. Plugins dialog will open.
5. Navigate to " plugins"  and observe colour contrast ratio.

**Expected Result:**
Colour contrast ratio should be greater than or equal to 4.5:1 for selected plugins .

**Actual Result:**
Colour contrast ratio is  4.4:1 for selected plugins .
**Platform:**
IntelliJ IDEA 2017.3.2 (Community edition)
Build# IC-173.4127.27
JRE:1.8.0_152-release-1024-b8amd64
JVM: OpenJDK 64 Bit Server VM by Jetbrains.
**OSBuild Version:** Version 1709(Os Build 16299.192)
**CCA version:** 2.5.0.0

![cca for plugins is plugins window in settings](https://user-images.githubusercontent.com/22956130/35681409-2799c8d2-0712-11e8-9f00-b004fef19d5b.png)

**Comment by Eskibear** [comment]
duplicated with #1372 

And it's out of the toolkit's scope.

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749856
